spec/unit/plugins/linux/network_spec.rb

```diff
@@ -1219,6 +1219,38 @@
         end
       end
 
+      describe "when there're multipath routes" do
+        let(:linux_ip_route) { "" }
+
+        let(:linux_ip_route_inet6) do
+          <<~EOM
+            default proto ra metric 1024 expires 1797sec \\        nexthop via fe80::c018:50ff:fe41:ca33 dev eth0.11 weight 1 \\        nexthop via fe80::c018:50ff:fe41:ca33 dev eth0 weight 1 pref medium
+            default proto ra metric 1050 expires 1796sec \\        nexthop via fe80::c018:50ff:fe3f:9184 dev eth3 weight 1 \\        nexthop via fe80::c018:50ff:fe3f:9184 dev eth13 weight 1 pref medium
+          EOM
+        end
+
+        before do
+          plugin.run
+        end
+
+        it "completes the run" do
+          expect(plugin.logger).not_to receive(:trace).with(/Plugin linux::network threw exception/)
+          expect(plugin["network"]).not_to be_nil
+        end
+
+        it "sets default ipv6 interface and gateway" do
+          expect(plugin["network"]["default_inet6_interface"]).to eq("eth0")
+          expect(plugin["network"]["default_inet6_gateway"]).to eq("fe80::c018:50ff:fe41:ca33")
+        end
+
+        it "sets the routes properly" do
+          expect(plugin["network"]["interfaces"]["eth0"]["routes"]).to eq([Mash.new( destination: "default", family: "inet6", proto: "ra", "via": "fe80::c018:50ff:fe41:ca33", "metric": "1024" )])
```

Comment:
be consistent on your hash syntax
```suggestion
          expect(plugin["network"]["interfaces"]["eth0"]["routes"]).to eq([Mash.new( destination: "default", family: "inet6", proto: "ra", via: "fe80::c018:50ff:fe41:ca33", metric: "1024" )])
```